src/vscode-bicep-ui/apps/resource-type-explorer/index.html

```diff
@@ -12,166 +19,155 @@
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachines",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachines/extensions",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachineScaleSets",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType:"virtualMachineScaleSets/virtualMachines",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachineScaleSets/virtualMachines/extensions",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachineScaleSets/networkInterfaces",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType:  "virtualMachineScaleSets/virtualMachines/networkInterfaces",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachineScaleSets/publicIPAddresses",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/operations",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/vmSizes",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/runCommands",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/systemInfo",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/virtualMachines",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/virtualMachineScaleSets",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType:"locations/publishers",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "operations",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachines/runCommands",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "virtualMachineScaleSets/applications",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType:  "virtualMachines/VMApplications",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/edgeZones",
+              apiVersion: "@2022-09-01"
+            },
+            {
+              resourceType: "locations/edgeZones/vmimages",
+              apiVersion: "@2022-09-01"
```

Comment:
Could we remove the `@` symbol from apiVersion? It's used as a separator in Bicep code, but we can achieve the same distinction in the resource type explorer UI by using padding.